File: SiriusQuality/SiriusQuality-Penman/src/cpp/SiriusQuality_Penman/PotentialET_PenmanRate.cpp
#include "PotentialET_PenmanRate.h"
using namespace SiriusQuality_Penman;


PotentialET_PenmanRate::PotentialET_PenmanRate() {}

double PotentialET_PenmanRate::getevapoTranspirationPenman() { return this->evapoTranspirationPenman; }
double PotentialET_PenmanRate::getevapoTranspirationPriestlyTaylor() { return this->evapoTranspirationPriestlyTaylor; }

void PotentialET_PenmanRate::setevapoTranspirationPenman(double _evapoTranspirationPenman) { this->evapoTranspirationPenman = _evapoTranspirationPenman; }
void PotentialET_PenmanRate::setevapoTranspirationPriestlyTaylor(double _evapoTranspirationPriestlyTaylor) { this->evapoTranspirationPriestlyTaylor = _evapoTranspirationPriestlyTaylor; }

File: SiriusQuality/SiriusQuality-Penman/src/java/SiriusQuality_Penman/PotentialET_PenmanRate.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;

public class PotentialET_PenmanRate
{
    private double evapoTranspirationPenman;
    private double evapoTranspirationPriestlyTaylor;
    
    public PotentialET_PenmanRate() { }
    
    public PotentialET_PenmanRate(PotentialET_PenmanRate toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.evapoTranspirationPenman = toCopy.getevapoTranspirationPenman();
            this.evapoTranspirationPriestlyTaylor = toCopy.getevapoTranspirationPriestlyTaylor();
        }
    }
    public double getevapoTranspirationPenman()
    { return evapoTranspirationPenman; }

    public void setevapoTranspirationPenman(double _evapoTranspirationPenman)
    { this.evapoTranspirationPenman= _evapoTranspirationPenman; } 
    
    public double getevapoTranspirationPriestlyTaylor()
    { return evapoTranspirationPriestlyTaylor; }

    public void setevapoTranspirationPriestlyTaylor(double _evapoTranspirationPriestlyTaylor)
    { this.evapoTranspirationPriestlyTaylor= _evapoTranspirationPriestlyTaylor; } 
    
}